src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,47 @@
+package duke.task;
+
+/**
+ * Encapsulation of a to-do.
+ *
+ * @author Sun Ruoxin
+ */
+public class Todo extends Task {
+    /**
+     * Class constructor.
+     *
+     * @param description the description of the to-do
+     */
+    public Todo(String description) {
+        super(description);
+    }
+
+    /**
+     * Class constructor with specified status.
+     *
+     * @param description the description of the to-do
+     * @param isDone the status of the to-do
+     */
+    public Todo(String description, boolean isDone) {
+        super(description, isDone);
+    }
+
+    /**
+     * Returns the type of the task.
+     *
+     * @return "T" representing to-do
+     */
+    @Override
+    public String getType() {
```

Comment:
I think this could be a static variable instead, since all Todo classes share the same Type, which is 'T'.

Reply:
Hi, thanks for your suggestion! I do get your point on how all Todo instances return the same 'T' thus the method should be static. However, since the getType() methods in Todo, Deadline and Event are overwriting the abstract method in Task, they cannot be static as overriding depends on dynamic binding:(